### ISSUE TITLE
Works for Handelsbanken B stock, but not for Handelsbanken A stock

### ISSUE THREAD

**Issue by MichaelBrimer** [custom-components/sensor.avanza_stock#175]
This is my configuration
sensor:
  - platform: avanza_stock
    stock:
      - id: 5264
        name: Handelsbanken

It works for 5265, but does not works for 5264
5264 is Handelsbanken A, and 5265 is Handelsbanken B

**Comment by claha** [comment]
Just tried it and I had no problems. What version og home assistant and avanza_stock. Can you provid som logs or something

**Comment by MichaelBrimer** [comment]
Hi
I have played with it and now I can get it to work.
Looking in "Developer tools", states, entity, I realised that for some strange reason, I had sensor.handelsbanken, sensor.handelsbanken_2, sensor.handelsbanken_3.
I do not understand this, it seems as it is some kind of cashed stuff?
Anyway, then I tried with this configuration and it now works.
      - id: 5264
        name: banken
Thank you for your time, and your help :-)
